Update README.md

DurvalMenezes opened the pull request:
Renamed "Support" section to "Prerequisites", added tip about nvidia-docker in the "Troubleshooting and  FAQ" section.

DurvalMenezes committed:
Update README.md

Renamed "Support" section to "Prerequisites", added tip about nvidia-docker in the "Troubleshooting and  FAQ" section.

stefancrain commented on README.md:
```diff
@@ -23,7 +23,7 @@ TL;DR: **Folding at home** is simulating the dynamics of COVID-19 proteins to hu
 I'm not a member of the F@H team. I actively work on issues here, and have set this up to [automatically update](.github/workflows/schedule.yml) in my absence. This image is published to [stefancrain/folding-at-home](https://hub.docker.com/repository/docker/stefancrain/folding-at-home) and automatically updated as the folding-at-home team releases changes.
 
-## Support
+## Prerequisites
```
👍

DurvalMenezes replied:
Great!

stefancrain commented on README.md:
```diff
@@ -76,6 +76,10 @@ TODO: create k8s cluster and test!
 
 ## Troubleshooting and FAQ
 
+### nvidia-docker 
```
A couple of thoughts here : 

- The content of this section does not follow the FAQ pattern of the readme, All of the sections begin with questions. 
- It closely resembles the nvidia-docker blurb in Prerequisites (but without my guide), Please work to further differentiate it from the prereqs blurb. 
- The url for `(here)[https://github.com/NVIDIA/nvidia-docker]` is not displaying correctly as you have the brackets backward, (here)[https://github.com/NVIDIA/nvidia-docker] vs [here](https://github.com/NVIDIA/nvidia-docker)

DurvalMenezes replied:
Thanks for the suggestions, and sorry about the stupid mistake re: url.
I think my latest commit nails it down; please have a look.

stefancrain reviewed (changes requested):
Thanks so much for the PR! I've included a couple of requests for formatting changes :-)

DurvalMenezes committed:
Implemented formatting changes as requested

Rephrased as a question, referenced 'Prerequisites" section instead of (badly) trying to repeat it.

DurvalMenezes committed:
Reformatted...

...for the same appearance as the other questions (ie, question as a quote, and answer following in normal body type).

stefancrain reviewed (approved):
👍

DurvalMenezes commented:
Thanks for the speedy response!
